sys/linux/vnet.txt

```diff
@@ -508,19 +510,40 @@ ipv6_addr_multicast2 {
 } [packed]
 
 ipv6_addr [
-	rand_addr	array[int8, 16]
+# Few public random addresses
+	rand_addr	ipv6_addr_random
 	empty		ipv6_addr_empty
 # These correspond to LOCAL_IPV6/REMOTE_IPV6/DEV_IPV6 in executor/common_linux.h
 	local		ipv6_addr_t[const[0xaa, int8]]
 	remote		ipv6_addr_t[const[0xbb, int8]]
 	dev		ipv6_addr_t[netdev_addr_id]
 	initdev		ipv6_addr_initdev
+# Some special addresses:
 	loopback	ipv6_addr_loopback
 	ipv4		ipv6_addr_ipv4
 	mcast1		ipv6_addr_multicast1
 	mcast2		ipv6_addr_multicast2
+# Several custom private ranges with 2 addresses each: fc0X::/127.
+# Can be used for custom things within the tested image.
+	private0	ipv6_addr_private[0]
+	private1	ipv6_addr_private[1]
+	private2	ipv6_addr_private[2]
 ] [size[16]]
 
+ipv6_addr_random {
```

Comment:
Other `ipv6_addr_` structs are above `ipv6_addr`, I think we should keep them all together.

Reply:
Done.